src/inverseRobot.js

```diff
@@ -24,7 +24,19 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+  const keys = Object.keys(robot);
+  const values = Object.values(robot);
+
+  for (let i = 0; i <= values.length - 1; i++) {
+    if (values[i] in newRobot) {
+      return null;
+    }
+
+    newRobot[values[i]] = keys[i];
```

Comment:
here `values[i]` is repeated, would be great to create variables for repeated values
```
const key = values[i]
const value = keys[i]
```

or while using Object.entries:
```
cosnt [value, key] = entry;
```